app/services/xccdf_report_parser.rb

```diff
@@ -44,6 +44,13 @@ def save_all
     Host.transaction do
       save_all_benchmark_info
       save_all_test_result_info
+      import_remediations
```

Comment:
I understand importing remediations after every datastream import, but why after every report import?

Reply:
Well, tbh I think the QA pipeline uploads reports without ever importing the datastream first, and this is certainly a possibility in our app as well if someone uploads a report from a datastream we don't know about already. When this happens, no remediations are found in the frontend/from the perspective of the user. It seems this could be fixed by setting `remediation_available` to true/false if cached or nil if it's not cached.